uweb3/pagemaker/__init__.py

```diff
@@ -396,8 +396,9 @@ def Static(self, rel_path):
       Page: contains the content and mimetype of the requested file, or a 404
             page if the file was not available on the local path.
     """
-    rel_path = os.path.abspath(os.path.join(os.path.sep, rel_path))[1:]
-    abs_path = os.path.join(self.PUBLIC_DIR, rel_path)
+
```

Comment:
This would allow path traversal to Above the PUBLIC_DIR.
When rel_path is for example '../../test' The previous code would add 'test' to the public_dir, resulting in public_dir+'test'.
Your code would result in public_dir+'../../test' which would leak the contents of the file test from two directories above our public dir. Some knowledge of how uweb works, or how the rest of the filesystem works and someone could steal any by uweb readable file's contents..

Reply:
Should be fixed now. Added a check that ensures target_dir is inside PUBLIC_DIR.